docs/.markdownlint-cli2.js

```diff
@@ -116,7 +116,8 @@ module.exports = {
                 "Microsoft",
                 "npm",
                 "Routerlicious",
-                "Tinylicious"
+                "Tinylicious",
+                "tinylicious.md", // hack
```

Comment:
Could you call out the hack here? Why do we need to starts specifying ".md" suffix  in the other files?